OneCommunityGlobal/HighestGoodNetworkApp
# Yan - Summary Management Group with CRUD PARTIAL PR, CHECK BUT DONT’ MERGE

CRUD for SummaryManagement Page / Group.

This PR is for CRUD for summary management group. Please also **push the backend** pr  Yanxu/crud to test it work or not

how to test it 
1) Add Summary Group
<img width="353" alt="image" src="https://user-images.githubusercontent.com/108507783/229259576-ababff98-e3f9-4823-9b63-b9b485269849.png">

click the create summary group  and input the name of summary group 
<img width="600" alt="image" src="https://user-images.githubusercontent.com/108507783/229259617-2b12c53c-b5d9-4299-b61d-958ccfce1620.png">

click the ok button and then check whether it show on table
2) Delete Summary Group
<img width="845" alt="image" src="https://user-images.githubusercontent.com/108507783/229259676-c58d76c0-a4bb-4537-bc1b-a1f0fd5237d0.png">
click the delete button and click the delete button on pop up to check whether the summary group has been deleted 
3) Update Summary Group
<img width="882" alt="image" src="https://user-images.githubusercontent.com/108507783/229259709-dc374bae-8031-4751-a9b8-fcda0a861046.png">
click the edit button and input the new name of summary group  to check it update on the table successfully 
<img width="465" alt="image" src="https://user-images.githubusercontent.com/108507783/229259765-e629cddc-38eb-493f-9f51-64befbfd2732.png">

4) Set Inactive for summary Group 
click the delete button  and click set inactive button  after click it, please check if the green button change to grey button for active column 
<img width="490" alt="image" src="https://user-images.githubusercontent.com/108507783/229259851-93d3a7e0-f37f-48ae-b0f9-9704589d47da.png">
5. Set the summary group active or inactive 
<img width="909" alt="image" src="https://user-images.githubusercontent.com/108507783/229259936-8e8c9306-6632-42bf-9fbf-cbc9f7a72e66.png">
go the active column , click the green button or grey button
<img width="607" alt="image" src="https://user-images.githubusercontent.com/108507783/229259969-00423923-8f2d-4f2c-a2c5-dd45a7a71130.png">
it should be active or inactive after clicking.
